### ISSUE TITLE
Do we really need chapter descriptions this lengthy? 

### ISSUE THREAD

**Issue by Shadowtail117** [Shadowtail117/Shadows-Parallel#16]
## We should truncate chapter descriptions to be short, concise summaries of the chapter, not rewordings of the scene descriptions.
For most of the chapters, the chapter description and the scene description appear to be almost the same except for the fact that the scene description is typically more detailed. Even now as I'm writing the chapter descriptions for the latest chapters I'm finding that I'm merely rewording a lot of the scene descriptions; it's redundant.
What would A. Reduce file size, B. Create conciseness (after all, reading the same thing twice when reading the outline? that is just silly), and C. Advance development, is making the chapter descriptions much, much shorter; **as in a maximum of ~5 sentences.** This ensures that someone can simply take a glance at a chapter description and know the general gist of what is occurring in that time period, rather than have to read a couple paragraphs' worth.
In short, minimizing the size of the paragraphs and leaving the chunky bits to scene descriptions is the most effective way to progress; it will reduce filesize (which isn't really an issue but is still a good thing nonetheless), make the outline more concise, and allow us to allocate more time into more pertinent things, like rewriting chapters 1-7  in #14 .

**Comment by Shadowtail117** [comment]
466b7e9a9d0cab13f9ff601142091d4f55ab0131 looks pretty good.

**Comment by Shadowtail117** [comment]
Forgot to close this when I did #18 .
